execute/group_lookup.go

```diff
@@ -295,3 +297,153 @@ func (l *GroupLookup) Range(f func(key flux.GroupKey, value interface{})) {
 		}
 	}
 }
+
+// RandomAccessGroupLookup is a GroupLookup container that is optimized
+// for random access.
```

Comment:
Looking at the source, I'm not sure what the "random access" part of this is. Is it a function of `map`?

Reply:
Yea. It's meant for areas where we don't necessarily know the order of incoming points where it will work consistently no matter the order they get added.